anaconda3/arm64/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM arm64v8/amazonlinux:2.0.20210421.0
```

Comment:
Shouldn't that file be located in the folder `./anaconda3/amazonlinux/`, as we also have `./anaconda3/{debian, alpine}/` ... that would allow also to later do multich arch images with docker buildx.